docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -522,6 +522,8 @@ following properties:
   - Enable view endpoints. Defaults to `true`.
 * - `iceberg.rest-catalog.signing-name`
   - AWS SigV4 signing service name. Defaults to `execute-api`.
+* - `iceberg.rest-catalog.google-project-id`
+  - Google Cloud project name. Example: `development-123456`.
```

Comment:
Worth saying here that's applicable when `iceberg.rest-catalog.security=google`?